omod/src/main/webapp/resources/scripts/account/changePassword.js

```diff
@@ -27,12 +27,21 @@ MatchedInputFieldValidator.prototype = new FieldValidator();
 MatchedInputFieldValidator.prototype.constructor = MatchedInputFieldValidator;
 MatchedInputFieldValidator.prototype.validate = function(field) {
 
-    var otherField = jQuery('#'+field.element.attr('matched-field-id'));
+
+    var newPassword = jQuery("#newPassword").val();
```

Comment:
Why did you change this? This should be a generic field validator that is not aware of the field ids because they can be set to anything. Can you please remove these changes?